src/gam.py

```diff
@@ -7970,21 +7970,21 @@ def _getBuildingAttributes(args, body={}):
     else:
       print u'ERROR: %s is not a valid argument for "gam create|update building"' % myarg
       sys.exit(3)
-  return body
 
 def doCreateBuilding():
   cd = buildGAPIObject(u'directory')
-  body = {u'floorNames': [u'1'],
+  body = {u'floorNames': [u'1',],
           u'buildingId': unicode(uuid.uuid4())}
-  body = _getBuildingAttributes(sys.argv[3:], body)
+  _getBuildingAttributes(sys.argv[3:], body)
   print u'Creating building %s...' % body[u'buildingId']
   callGAPI(cd.resources().buildings(), u'insert',
            customer=GC_Values[GC_CUSTOMER_ID], body=body)
 
 def doUpdateBuilding():
   cd = buildGAPIObject(u'directory')
   buildingId = sys.argv[3]
-  body = _getBuildingAttributes(sys.argv[4:])
```

Comment:
I understand the behavior is the same here but the assignment ( body = ) makes it obvious that the function is actually modifying body. Let's stick with that form here.